avocado/utils/network/interfaces.py

```diff
@@ -571,3 +571,29 @@ def restore_slave_cfg_file(self):
                     os.remove(slave_config)
             except Exception as ex:
                 raise NWException(f"Could not restore the config file {ex}")
+
+    def are_packets_lost(self, peer_ip, count, option=None, sudo=False):
```

Comment:
For consistency, "option" should be "options" since we can pass multiple options here as described on the docstring.

Reply:
Also, what happens if a user call like this?

```python
are_packets_lost('127.0.0.1', 10, '-c 20')
```

Based on the docstring, give us the impression that we support both. I really didn't get why having count as a separated argument, since we have already 'options'.